tool/teleport/common/teleport.go

```diff
@@ -179,6 +182,7 @@ func Run(options Options) (app *kingpin.Application, executedCommand string, con
 	appStartCmd.Flag("debug", "Enable verbose logging to stderr.").Short('d').BoolVar(&ccf.Debug)
 	appStartCmd.Flag("pid-file", "Full path to the PID file. By default no PID file will be created.").StringVar(&ccf.PIDFile)
 	appStartCmd.Flag("auth-server", fmt.Sprintf("Address of the auth server [%s].", defaults.AuthConnectAddr().Addr)).StringsVar(&ccf.AuthServerAddr)
+	appStartCmd.Flag("proxy-server", fmt.Sprintf("Address of the proxy server [%s].", defaults.AuthConnectAddr().Addr)).StringsVar(&ccf.ProxyServerAddr)
```

Comment:
Don't use the auth connect addr in the help message, as the presence of auth port 3025 will confuse users (prefer 3080 or 443).